virt/virt-web-console-overview.adoc

```diff
@@ -623,7 +628,214 @@ You can create, edit, and clone VirtualMachine templates on the Templates page.
+=====
+[cols="1,3a", options="header"]
+|====
+|Element |Description
+|*YAML* switch |Select *ON* to view your live changes in the YAML configuration file.
+|*Save* button |Save changes to the YAML file.
+|*Reload* button |Discard your changes and reload the YAML file.
+|*Cancel* button |Exit the *YAML* tab.
+|*Download* button |Download the YAML file to your local machine.
+|====
+=====
+
+
+[id="ui-template-details-scheduling_{context}"]
+==== Scheduling tab
+
+You can configure scheduling on the *Scheduling* tab.
+
+.Scheduling tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element |Description
+|*YAML* switch |Select *ON* to view your live changes in the YAML configuration file.
+
+|Node selector
+|Click the edit icon to add a label to specify qualifying nodes.
+
+|Tolerations
+|Click the edit icon to add a toleration to specify qualifying nodes.
+
+|Affinity rules
+|Click the edit icon to add an affinity rule.
+
+|*Descheduler* switch
+|Enable or disable the descheduler. The descheduler evicts a running pod so that the pod can be rescheduled onto a more suitable node.
+
+|Dedicated resources
+|Click the edit icon to select *Schedule this workload with dedicated resources (guaranteed policy)*.
+
+|Eviction strategy
+|Click the edit icon to select *LiveMigrate* as the VirtualMachineInstance eviction strategy.
+|====
+=====
+
+[id="ui-template-details-network-interfaces_{context}"]
+==== Network interfaces tab
+
+You can manage network interfaces on the *Network interfaces* tab.
+
+.Network interfaces tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element |Description
+|*YAML* switch |Select *ON* to view your live changes in the YAML configuration file.
+|*Add network interface* button |Add a network interface to the template.
+|*Filter* field |Filter by interface type.
+|Search field |Search for a network interface by name or by label.
+|*Network interface* table |List of network interfaces.
+
+Click the Options menu {kebab} beside a network interface to select *Edit* or *Delete*.
+|====
+=====
+
+[id="ui-template-details-disks_{context}"]
+==== Disks tab
+
+You can manage disks on the *Disks* tab.
+
+.Disks tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element |Description
+|*YAML* switch |Select *ON* to view your live changes in the YAML configuration file.
+|*Add disk* button |Add a disk to the template.
+|*Filter* field |Filter by disk type.
+|Search field |Search for a disk by name.
+|*Disks* table |List of template disks.
+
+Click the Options menu {kebab} beside a disk to select *Edit* or *Detach*.
+|====
+=====
+
+[id="ui-template-details-scripts_{context}"]
+==== Scripts tab
+
+You can manage the cloud-init settings, SSH keys, and Sysprep answer files on the *Scripts* tab.
+
+.Scripts tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element |Description
+|*YAML* switch |Select *ON* to view your live changes in the YAML configuration file.
```

Comment:
```suggestion
|*YAML* switch |Set *ON* to view your live changes in the YAML configuration file.
```

Reply:
You don't need to comment every time the same error occurs. Just leave one comment that says "Global: etc". It's a lot less work for both of us.